cg/cli/workflow/commands.py

```diff
@@ -350,3 +354,30 @@ def microsalt_past_run_dirs(
 
     context.obj.meta_apis["analysis_api"]: MicrosaltAnalysisAPI = MicrosaltAnalysisAPI(context.obj)
     context.invoke(past_run_dirs, yes=yes, dry_run=dry_run, before_str=before_str)
+
+
+@click.command("nf-workflow-past-run-dirs")
```

Comment:
I don't see why we can't reuse `@click.command("past-run-dirs")` for this. If needed we could move the logic to the analysis API and if some things are different we can  create additional methods in the NF workflow specific APIs